src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeHttpWrapperTests.cs

```diff
@@ -136,6 +136,67 @@ public async Task PostAsync_Exception_ExceptionNotCaught()
             await act.Should().ThrowAsync<NotImplementedException>();
         }
 
+        [TestMethod]
+        public async Task GetAsync_ExecutesRequestsOnTheUrl()
```

Comment:
These three tests are copies of the equivalent POST ones, but with `content` argument removed.